pkg/search/result.go

```diff
@@ -220,6 +221,12 @@ func (issue Issue) ExportData(fields []string) map[string]interface{} {
 				"name":          name,
 				"nameWithOwner": nameWithOwner,
 			}
+		case "state":
+			if !issue.PullRequest.MergedAt.IsZero() {
```

Comment:
working on this PR I learned about [`time.IsZero()`](https://pkg.go.dev/time#Time.IsZero) 🎉